sites/partners/lib/helpers.ts

```diff
@@ -109,6 +113,17 @@ export const getRentType = (unit: TempUnit): string | null => {
     : null
 }
 
+export const getRentTypeFromUnitsSummary = (summary: TempUnitsSummary): string | null => {
```

Comment:
I think I'm missing something, but where is this called?

Also, I think this would feel more readable to me if we use normal if-else instead of the ternary if